static/js/src/formValidation.js

```diff
@@ -164,9 +164,10 @@ afterSubmit = function(download_asset_url, return_url, isModal, thankYouMessage,
 
 // recaptcha submitCallback
 CaptchaCallback = function() {
-  let recaptchas = document.querySelectorAll("div[class^=g-recaptcha]");
+  let recaptchas = [].slice.call(document.querySelectorAll("div[class^=g-recaptcha]"));
```

Comment:
can this `CaptchaCallback` function be moved to a common file - it's copy-pasta'd

Reply:
yes. But not in this PR. It should be part of a refactor of that part of the code.